src/main/java/io/vertx/ext/camel/CamelBridgeOptions.java

```diff
@@ -0,0 +1,75 @@
+/*
+ *  Copyright (c) 2011-2015 The original author or authors
+ *  ------------------------------------------------------
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  and Apache License v2.0 which accompanies this distribution.
+ *
+ *       The Eclipse Public License is available at
+ *       http://www.eclipse.org/legal/epl-v10.html
+ *
+ *       The Apache License v2.0 is available at
+ *       http://www.opensource.org/licenses/apache2.0.php
+ *
+ *  You may elect to redistribute this code under either of these licenses.
+ */
+package io.vertx.ext.camel;
+
+import org.apache.camel.CamelContext;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Camel bridge configuration.
+ *
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ */
+public class CamelBridgeOptions {
```

Comment:
even though it's not polyglot, we should annotation with @DataObject (just in case) and it enforces conventions in our data objects and it generates cheatsheet doc